packages/psss-server/src/connections/MeditrakConnection.ts

```diff
@@ -103,7 +120,8 @@ export class MeditrakConnection extends ApiConnection {
     surveyCode: string,
     organisationUnitCode: string,
     period: string,
-    answers: Record<string, number>,
+    answers: Answers,
+    surveyResponseId?: string | undefined,
```

Comment:
Yeah the `changes` endpoint unfortunately only returns a success message. What you suggested also works too. I did this because just in case in the future there's something useful that we want to get from the response of `changes` endpoint. Or we can return both the generated id and the response of `changes` endpoint